### ISSUE TITLE
The basic conceptions of TensorFlow using Mnist model

### ISSUE THREAD

**Issue by LeeJMSSS** [LeeJMSSS/constructionOfMajor#3]
==============================================================
 소프트맥스 회귀는 두 단계로 이루어짐
입력한 데이터가 각 클래스에 속한다는 증거를 수치적으로 계산
그 이후 계산한 값을 확률로 변환하는 것.

 한 이미지가 특정 클래스에 속하는지 계산하기 위해서는 가중치합을 한다.
그 이미지에 가까우면 양의 값, 반하는 것이라면 음의 값.

이 때 새로운 클래스가 생길 수 있으므로 바이어스를 추가적으로 계산함.(?)

x= tf.placeholder(tf.float32,[None,784])

784차원의 벡터로 변형된 mnist 이미지의 데이터

w= tf.Variable(tf.zeros([784.10]))
784차원의 이미지 벡터를 곱해서 클래스에 대한 증거값을 나타내는 10차원 벡터를 얻고자 함.

b= tf.Variable(tf.zeros([10]))
10차원 벡터에 더하기 위해서 [10]의 형태

y= tf.nn.softmax(tf.matmul(x,W)+b)

텐서플로우는 모델 자체가 유연해서 어디서나 모델 실행가능

모델이 좋다 안좋다 판단 = cost 와 loss 값으로 판단
모델 손실 정의를 위해 자주 사용되는 것 = 크로스 엔트로피

모델 자체를 생성한다?????(무슨말일까)
